rust/agama-lib/src/storage/client.rs

```diff
@@ -69,6 +70,11 @@ impl<'a> StorageClient<'a> {
                 .cache_properties(zbus::CacheProperties::No)
                 .build()
                 .await?,
+            // Same than above, actions are reexported with every call to recalculate
+            devices_proxy: DevicesProxy::builder(&connection)
+                .cache_properties(zbus::CacheProperties::No)
```

Comment:
The *Storage1.Devices* interface is not reexported. This should be similar to the *storage_proxy* and *calculator_proxy*. I would say caching it is ok here. Anyway, it is not that important because we are caching at client level.

Reply:
At some point I would like to reevalute the *Devices* interface. In think it would be better to add a new method *Storage1#devices* instead of having a separate interface. That method would report a JSON according to a schema, for example:

```
{
  system: {},
  target: {},
  actions: {}
}
```

Reply:
Yes. Definitely all the proxies need to be reconsidered... afterwards.